.github/workflows/pipeline.yml

```diff
@@ -26,3 +26,23 @@ jobs:
       - uses: actions/setup-go@v2
       # use go cli
       - run: go version
+      - run: go build *.go
```

Comment:
have to move the go files to root since the action we are using to build the image directly into Digital Ocean Registry has some missing features see: https://github.com/ripplr-io/docr-docker-publish/pull/2